internal/config/logger.go

```diff
@@ -21,6 +21,7 @@ type Logger struct {
 	FullScreenLogs bool  `yaml:"fullScreenLogs"`
 	TextWrap       bool  `yaml:"textWrap"`
 	ShowTime       bool  `yaml:"showTime"`
+	UseUTC         bool  `yaml:"useUTC"`
```

Comment:
Best to reverse this so we can use the zero value as the default ie UseLocalTime